src/models/signature-chain-link.ts

```diff
@@ -0,0 +1,21 @@
+import { prop } from "@typegoose/typegoose";
+import Form from "./form";
+import { SigningPositions } from "@/enums/signing-positions";
+import Position from "./position";
+
+class SignatureChainLink {
+
+    @prop({ required: true, ref: () => Form, type: () => Form })
+    public form!: Form;
```

Comment:
I don't think this field is necessary anymore - we're never going to try and look up the form from this model, only the other way around